[2025-06-15T10:27:10.710+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: retail_analytics_airflowDAG.load manual__2025-06-15T10:26:11.682373+00:00 [queued]>
[2025-06-15T10:27:10.720+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: retail_analytics_airflowDAG.load manual__2025-06-15T10:26:11.682373+00:00 [queued]>
[2025-06-15T10:27:10.720+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2025-06-15T10:27:10.735+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load> on 2025-06-15 10:26:11.682373+00:00
[2025-06-15T10:27:10.741+0000] {standard_task_runner.py:60} INFO - Started process 204 to run task
[2025-06-15T10:27:10.743+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'retail_analytics_***DAG', 'load', 'manual__2025-06-15T10:26:11.682373+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline_dag.py', '--cfg-path', '/tmp/tmpj9jdnnp6']
[2025-06-15T10:27:10.746+0000] {standard_task_runner.py:88} INFO - Job 30: Subtask load
[2025-06-15T10:27:10.764+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-15T10:27:10.793+0000] {task_command.py:423} INFO - Running <TaskInstance: retail_analytics_airflowDAG.load manual__2025-06-15T10:26:11.682373+00:00 [running]> on host 6843dfc50be4
[2025-06-15T10:27:10.851+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='retail_analytics_***DAG' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-06-15T10:26:11.682373+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-15T10:26:11.682373+00:00'
[2025-06-15T10:27:10.853+0000] {data_load_to_sql.py:25} INFO - Started loading...
[2025-06-15T10:27:29.898+0000] {logging_mixin.py:188} INFO - ✅ Table 'sales_gold' exists. Data appended.
[2025-06-15T10:27:30.051+0000] {logging_mixin.py:188} INFO - ✅ Table 'customers_gold' exists. Data appended.
[2025-06-15T10:27:30.117+0000] {logging_mixin.py:188} INFO - ✅ Table 'inventory_gold' exists. Data appended.
[2025-06-15T10:27:30.118+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-15T10:27:30.131+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=retail_analytics_***DAG, task_id=load, execution_date=20250615T102611, start_date=20250615T102710, end_date=20250615T102730
[2025-06-15T10:27:30.200+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-15T10:27:30.215+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
